src/display/text_layer.js

```diff
@@ -499,9 +501,12 @@ var renderTextLayer = (function renderTextLayerClosure() {
    * @param {boolean} enhanceTextSelection
    * @private
    */
-  function TextLayerRenderTask(textContent, container, viewport, textDivs,
-                               enhanceTextSelection) {
+  function TextLayerRenderTask({ textContent, textContentStream, container,
+                                 viewport, textDivs, enhanceTextSelection, }) {
+    this.textContentItemsStr = [];
```

Comment:
Similar to another comment in `web/text_layer_builder.js`, I think this should be handled similar to the `textDivs` instead.